environments/staging/values.yaml

```diff
@@ -121,3 +121,6 @@ dreams:
 camps-index:
   enabled: true
   sparkAppSecretName: sparkdb-app
+
+suppliers:
+  bucket: midburn-spark-supplier-files
```

Comment:
this value will only be visible to `suppliers` chart, it won't be accessible for `spark` chart